src/stm32/stm32f0.c

```diff
@@ -104,7 +104,8 @@ pll_setup(void)
     if (CONFIG_CLOCK_REF_8M) {
         // Configure 48Mhz PLL from external 8Mhz crystal (HSE)
         RCC->CR |= RCC_CR_HSEON;
-        cfgr = RCC_CFGR_PLLSRC_HSE_PREDIV | ((6 - 2) << RCC_CFGR_PLLMUL_Pos);
+        cfgr = RCC_CFGR_PLLSRC_HSE_PREDIV | ((6 - 2) << RCC_CFGR_PLLMUL_Pos)
+             | RCC_CFGR_ADCPRE_DIV2;
```

Comment:
The stm32f070 reference manual says that the ADCPRE bit is obsolete.  It seems to recommend setting HSI14ON in RCC_CR2.

Reply:
Removed obsolete line.  Changed ADC from system clock to internal 14mhz clock.